Bug 1303299, CLI tools download and About page for web console

ahardin-rh opened the pull request:
https://bugzilla.redhat.com/show_bug.cgi?id=1303299

ahardin-rh commented:
@adellape PTAL

tpoitras commented:
I think there's gonna be a clash with this PR: https://github.com/openshift/openshift-docs/pull/1467

Whichever one merges first will force the other to rebase.

ahardin-rh commented:
Fixed the merge conflicts and reworked my edits.

adellape commented:
@ahardin-rh We're forgetting to take openshift-dedicated and openshift-online into account for get_started_cli.

adellape commented:
Actually, that needs to get fixed before the new About page stuff here, so I'll fix that in a separate PR to get published this week.

ahardin-rh commented:
In today's team meeting, it was suggested that I incorporate a screenshot. I will get that added soon.

tpoitras commented:
:camera: :sparkler: :+1:

adellape commented:
@ahardin-rh Just a reminder, this is unblocked now that the fix in https://github.com/openshift/openshift-docs/pull/1661 was merged/published.

ahardin-rh commented:
:construction_worker: Thanks! I'm on it! :hammer:

ahardin-rh commented:
@adellape Completed my latest edits. Please check behind me to see if I missed anything.  :tada:

adellape commented on cli_reference/get_started_cli.adoc:
```diff
@@ -36,6 +36,13 @@ for instructions per your workstation's operating system.
 
 [[installing-the-cli]]
 == Installing the CLI
+You can access the *About* page on the web console
```
s/access/download the CLI from/

As it's written, the sentence sounds like it's just saying you can access the page but not what to do there, or that the page itself may or may not be enabled by the admin.

Suggestion: Maybe also start this with something like, "The easiest way to download the CLI is by accessing..."

adellape commented on cli_reference/get_started_cli.adoc:
```diff
@@ -36,6 +36,13 @@ for instructions per your workstation's operating system.
 == Installing the CLI
+You can access the *About* page on the web console
+ifdef::openshift-enterprise,openshift-origin[]
+if your cluster administrator has enabled it
```
s/it/the download links/

adellape commented on cli_reference/get_started_cli.adoc:
```diff
@@ -195,6 +222,13 @@ $ tar -xf <file>
+unpack the CLI from the
+link:../install_config/web_console_customization.html#adding-or-changing-links-to-download-the-cli[*About*
+page on the web console].
+endif::[]
```
I think this should be split up and put into the respective ifdefs like you have it in the Windows and Mac sections.

adellape replied:
Or, put this right before "Then, unpack the archive" a few lines up.

ahardin-rh commented:
@adellape Thanks! Made these updates and added in the new version of the screenshot with pixelation to both sections.

ahardin-rh commented:
[rev_history]
|link:../architecture/infrastructure_components/web_console.html[Web Console]

|Added a link:../architecture/infrastructure_components/web_console.html#web-console-cli-downloads[CLI Downloads] section, including a screenshot of the new *About* page. 
%
|link:../cli_reference/get_started_cli.html[Get Started with the CLI]

|In the link:../cli_reference/get_started_cli.html#installing-the-cli[Installing the CLI section], added information about downloading the CLI from the *About* page, including a screenshot of the *About* page.
%
|link:../install_config/web_console_customization.html[Customizing the Web Console]

|In the link:../install_config/web_console_customization.html#adding-or-changing-links-to-download-the-cli[Adding or Changing Links to Download the CLI] section, added information about downloading the CLI from the *About* page.
%
